lib/Model/Importer/Binary/PECOFFImporter.cpp

```diff
@@ -154,13 +160,92 @@ void PECOFFImporter::parseSymbols() {
   }
 }
 
+void PECOFFImporter::recordImportedFunction(
+  iterator_range<llvm::object::imported_symbol_iterator> Range,
+  uint32_t ImportAddressTableAddr) {
+  using namespace llvm::object;
+
+  // Index of entries within import table.
+  uint32_t Index = 0;
+  for (const ImportedSymbolRef &I : Range) {
+    StringRef Sym;
+    if (Error E = I.getSymbolName(Sym)) {
+      revng_log(Log, "Found an imported symbol without a name.");
+      continue;
+    }
+
+    // TODO: We may face some old linkers that use ordinal bits only
+    // so consider this info then.
+    uint16_t Ordinal;
+    if (Error E = I.getOrdinal(Ordinal)) {
+      revng_log(Log, "Found an imported symbol without an ordinal.");
+      continue;
+    }
+
+    if (Model->ImportedDynamicFunctions.count(Sym.str()))
+      continue;
+
+    // NOTE: This address will occur in the .text section as a target of a jump.
+    // Once we have the address of the entry within .idata, we can access
+    // the information about symbol.
+    MetaAddress AddressOfImportEntry = ImageBase + u64(ImportAddressTableAddr)
+                                       + u64(Index * CHAR_BIT);
```

Comment:
Can you elaborate on this? Using `CHAR_BIT` when computing an address is suspicious.

Reply:
Oh yes... I should have used the ```pointerSize``` instead...